### ISSUE TITLE
rename routine not considering ni, nj

### ISSUE THREAD

**Issue by markusritschel** [jbusecke/cmip6_preprocessing#50]
It seems like the renaming routine does not consider dimensions called `ni` and `nj`. 
They can be found, for example, in sea-ice related files, e.g. "siconc_SImon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc".
If I hand over an accordingly adjusted `rename_dict`, i.e. added "ni" and "nj" to the entries of "x" and "y":
```python
rename_dict = {
    # dim labels
    "x": ["x", "i", "nlon", "lon", "longitude", "ni"],
    "y": ["y", "j", "nlat", "lat", "latitude", "nj"],
    "lev": ["lev", "deptht", "olevel", "zlev", "olev"],
    "bnds": ["bnds", "axis_nbounds", "d2"],
    "vertex": ["vertex", "nvertex", "vertices"],
    # coordinate labels
    "lon": ["lon", "longitude", "nav_lon"],
    "lat": ["lat", "latitude", "nav_lat"],
    "lev_bounds": [
        "lev_bounds",
        "deptht_bounds",
        "lev_bnds",
        "olevel_bounds",
        "zlev_bnds",
    ],
    "lon_bounds": [
        "bounds_lon",
        "bounds_nav_lon",
        "lon_bnds",
        "x_bnds",
        "vertices_longitude",
    ],
    "lat_bounds": [
        "bounds_lat",
        "bounds_nav_lat",
        "lat_bnds",
        "y_bnds",
        "vertices_latitude",
    ],
    "time_bounds": ["time_bounds", "time_bnds"],
}

rename_cmip6(ds, rename_dict=rename_dict)
```
to the `rename_cmip6` routine, it doesn't change anything. Or am I missing something?

**Comment by jbusecke** [comment]
Thanks for raising the issue. This is very valuable input to make the package usable for more people (it is currently very focussed on ocean output, since that is what I primarily work with). 

You suggestion is spot on. 
Would you consider putting in a PR to fix this?

**Comment by jbusecke** [comment]
Closed via #51
